src/components/HomePage/HomepageTestimonials.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="testimonials">
+    <div layout="column u4">
+      <div v-for="t in testimonials" :key="t.name" layout="row u1">
+        <span flex
+          ><i>“{{ t.text }}”</i></span
+        >
+        <span flex
+          ><b>{{ t.name }}</b> ({{ t.institute }})</span
+        >
+      </div>
+    </div>
+    <p>
+      Cited in
+      <a
+        href="https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22quantum+game+with+photons%22&btnG="
```

Comment:
please encode the ampersands in URL. HTML attributes must be encoded.
```suggestion
        href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C5&amp;q=%22quantum+game+with+photons%22&amp;btnG="
```

Alternatively you can use vue-specific syntax and treat it as javascript string
```suggestion
        :href="'https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22quantum+game+with+photons%22&btnG='"
```